src/uwtools/cli.py

```diff
@@ -464,6 +466,17 @@ def _add_arg_cycle(group: Group) -> None:
     )
 
 
+def _add_arg_debug(group: Group) -> None:
+    group.add_argument(
+        _switch(STR.debug),
+        "--debug",
```

Comment:
I don't think this line is needed. For switches with short forms (e.g. `-v` for `--verbose`, this line would give the short form. But this one doesn't, and the long form is handled by the preceding line, which expands to the same thing.

Reply:
Ah good point - got this one cleaned out